src/libraries/System.Net.Security/src/System/Net/Security/SslCertificateTrust.cs

```diff
@@ -43,10 +42,18 @@ public static SslCertificateTrust CreateForX509Store(X509Store store, bool sendT
         [UnsupportedOSPlatform("windows")]
         public static SslCertificateTrust CreateForX509Collection(X509Certificate2Collection trustList, bool sendTrustInHandshake = false)
         {
+
+#if TARGET_WINDOWS
+            if (sendTrustInHandshake)
+            {
+                throw new PlatformNotSupportedException(SR.net_ssl_trust_collection);
+            }
+#else
```

Comment:
This doesn't really add value.  Windows is already not Linux and not macOS.  Same message, same truth table... let's get rid of preprocessor directives.

Reply:
Hm.  Actually, I see that this says "collection" where the existing says "handshake".

Are they really supposed to be different?  Why is Windows different than Android here?

Reply:
The difference is that it can't be supported on Windows (since there is no API which allows us to do it), while on other platforms, it may be the case that it just was not implemented yet.

There used to be this distinction before #65195, and I think that I should not have removed it.